### PR TITLE
Use native `bring_to_front` to focus Sublime's window

### PR DESCRIPTION
**Pull request by kaste**

### CODE REVIEW

File: FocusSublimeWindow.py [hunk 1]
```diff
@@ -4,13 +4,19 @@
 from sublime_plugin import WindowCommand
 
 
+NATIVE_FOCUS_WINDOW = int(sublime.version()) >= 4000
```
Comment (fregante) [inline FocusSublimeWindow.py]: It seems that it was introduced in BUILD 4113 specifically: https://www.sublimetext.com/download

BUILD 4107 also exists but does not have the method

```suggestion
NATIVE_FOCUS_WINDOW = int(sublime.version()) >= 4113
```

**Review by fregante — APPROVED** [review]
Thank you, this is excellent!

File: FocusSublimeWindow.py (hunk 1 above)
Comment (kaste) [reply FocusSublimeWindow.py]: 4107 is the first (stable) build that has the method.  (And 4067 ist the first *dev* build.)  4107 marks the v4, it's the initial v4. 

image

Comment (kaste) [inline FocusSublimeWindow.py]:
```suggestion
NATIVE_FOCUS_WINDOW = int(sublime.version()) >= 4067
```

**Comment by kaste** [comment]
Updated

**Comment by fregante** [comment]
Thanks!

**Comment by fregante** [comment]
Is this working for you? See the gif in https://github.com/fregante/GhostText/pull/253, ST is not being focused (I verified that it's the latest version)

**Comment by kaste** [comment]
It's not working here too atm.  (It *did* work though.)  On Windows Sublime's taskbar item flashes orange, so it indicates that ST wants the focus but Windows prevents the focus stealing.  

Reverting might be the best option as I'm not getting it to work in just a sec.  😖

**Comment by fregante** [comment]
Could it be a ST issue then?

**Comment by kaste** [comment]
I don't think so *if* ST is just *asking* for the focus.  At that time the textarea from the website (which is an input element) should have the (input) focus.  And without force, Windows should prevent the application switch.  (Because the user is about to type into the textarea.)  

I wonder why it did work but I also updated Windows (and brave) since the PR got merged.  I definitely can't get it to work right now but it did work back then.

**Comment by fregante** [comment]
The [API’s docs](https://www.sublimetext.com/docs/api_reference.html#sublime.Window.bring_to_front) mention:

> Bring the window in front of any other windows

If you have more context or time, could you ask ST’s dev? The focus part is pretty important in my view because otherwise you'd have to go find the window among the bunch, which almost defeats the purpose of GhostText 😰

**Comment by kaste** [comment]
I don't think ST should *force* the focus.  I thought the context/the "when" we call `bring_to_front()` would pass Windows checks and actually allow the focus switch.  

But it doesn't.  At least *most* of the time, because ironically just now I'm writing this text in ST and it *did* switch to ST automatically after `ctrl+shift+k`.  I think we need to call this when the textarea doesn't have focus.

**Comment by fregante** [comment]
> I don't think ST should force the focus. 

Then we can revert this PR, or maybe leave bring_to_front as "best effort" but still go ahead and call the other commands if they exist 

The intention of GhostText is clear: **switch to the editor**, so if it doesn't, it's just extra steps, every single time.

**Comment by kaste** [comment]
I would use `bring_to_focus` as a best effort and then do the other thing.  `nircmd` is not a real thing on Windows; on the other platforms this worked before probably.  

You didn't tell here that the extension itself has updates since this PR.  Have you bisected them?  Did you try this PR three weeks ago actually?

**Comment by kaste** [comment]
Changed my mind, see #5.

**Pull request merged**